src/main/java/com/stripe/model/EventData.java

```diff
@@ -10,6 +15,137 @@
 @Setter
 @EqualsAndHashCode(callSuper = false)
 public class EventData extends StripeObject {
+  // Original properties
   StripeObject object;
   Map<String, Object> previousAttributes;
+
+  // Augmented properties during deserialization
+  @Setter(AccessLevel.NONE)
+  JsonElement rawJson;
+  @Getter(AccessLevel.NONE)
+  @Setter(AccessLevel.NONE)
+  String apiVersion;
```

Comment:
When event data stands by itself, not tied to the root-level object, it should have information about its tagged api version to be useful for guiding the "safe" deserialization.
An alternative is to make `EventData` non-static inner class of `Event`, but it will have reference to all its parent attribute.